### PR TITLE
Add component type as heading to properties editor / node tooltip

### PR DESCRIPTION
**Pull request by marthacryan**
Fixes elyra-ai/elyra#1836. Adds component type to the top of the node properties editor so that user can see the component type being edited. 
image 
image

Also adds component type to the top of the tooltip:
image

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: packages/pipeline-editor/src/properties-panels/NodeProperties.tsx [hunk 1]
```diff
@@ -70,16 +83,19 @@ function NodeProperties({
...
-      onPropertiesUpdateRequested={onPropertiesUpdateRequested}
-      propertiesSchema={nodePropertiesSchema.app_data.properties}
-      onFileRequested={onFileRequested}
-      onChange={(data: any) => {
-        onChange?.(selectedNode.id, data);
-      }}
-      id={selectedNode.id}
-    />
+    <div>
+      <Heading> {nodePropertiesSchema.label} </Heading>
```
Comment (bourdakos1) [inline packages/pipeline-editor/src/properties-panels/NodeProperties.tsx:87]: Just remove this leading and trailing whitespace, otherwise looks good

**Review by ptitzler — APPROVED** [review]
LGTM based on screen captures. Did not try to build.

File: packages/pipeline-editor/src/PipelineEditor/index.tsx [hunk 2]
```diff
@@ -572,7 +572,14 @@ const PipelineEditor = forwardRef(
       if (isNodeTipEvent(tipType, e) && e.node.type === "execution_node") {
         const error = e.node.app_data.invalidNodeError;
         const properties = controller.current.properties(e.node.id);
-        return <NodeTooltip error={error} properties={properties} />;
+        const node: any = controller.current.getPaletteNode(e.node.op);
```
Comment (bourdakos1) [inline packages/pipeline-editor/src/PipelineEditor/index.tsx:575]: does this need to be `any`?

**Review by bourdakos1 — CHANGES_REQUESTED** [review]
Can we add a test that tooltip renders the header and same with the properties panel?

File: packages/pipeline-editor/src/PipelineEditor/index.tsx (hunk 2 above)
Comment (marthacryan) [reply packages/pipeline-editor/src/PipelineEditor/index.tsx]: It was typed as `unknown` so I couldn't access `node.label` - I just pushed a change that uses the typed `getAllPaletteNodes` function instead

**Comment by marthacryan** [comment]
@bourdakos1 How do the tests I added look?

**Review by bourdakos1 — APPROVED** [review]

**Pull request merged**